nkinsaul/tunes-and-brews
# Feature/remove from saved

1. What's this PR do?
    - This will add a button to the saved events view to remove an event from saved.
1. Where should the reviewer start?
    - SavedEvents.js
    - App.js
1. How should this be manually tested?
    - Run `npm start` and navigate to http://localhost:3000/
    - Click on an event and add to save
    - Go back to home, click on another event and add to saved.
    - Go to saved events and see the events you added
    - Click remove event button to remove the event from saved.
1. Any background context you want to provide?
    - Forgot to add this on the first addition.
    - Still need to add some logic to make sure an event can't be added twice.
1. What are the relevant tickets?
1. Screenshots (if appropriate)
1. Questions:
